templates/content/GeoBlazor.Template.Server/GeoBlazor.Template.Server.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <UserSecretsId>cb19df39-b8dd-47f7-91a7-f23655fdfde8</UserSecretsId>
```

Comment:
I know we started talking about this yesterday. I think it's a bad idea for us to set the `UserSecretsId` for a template. We should instead put the blank `ArcGISApiKey` setting in `appsettings.json` or `appsettings.Development.json` like in the Maui template, and add instructions to the ReadMe about using user secrets instead.

Here's a scenario why this matters. A user who has multiple apps started with this template would end up with shared user secrets, which they might not want.